fe/fe-core/src/main/java/com/starrocks/external/HiveMetaStoreTableUtils.java

```diff
@@ -162,6 +162,8 @@ public static boolean validateColumnType(String hiveType, Type type) {
                 }
                 return validateColumnType(hiveType.substring(hiveType.indexOf('<') + 1, hiveType.length() - 1),
                         ((ArrayType) type).getItemType());
+            case "BINARY":
```

Comment:
for validation, just return false for unsupported type, so you can remove this

Reply:
done